data/human/human missions.txt

```diff
@@ -9341,3 +9342,799 @@ mission "Timothy Radrickson 7: Grave"
+			`	"You stupid, selfish, arrogant brat. You just needed to do as you were told. To play your pathetic part. You couldn't could you? You were always worthless. If only your brother had survived." Deoric picks up a broken furniture leg and confidently wields it like a club.`
+			`	At this Ildico's wicked grin widens deeper. "My brother? Dear Hogne. Father, I killed him. He couldn't handle the ice, the isolation, the hopelessness of Hope. He lost his mind. He tried to make me his wife. And mother? She cracked long before that. An entitled debutant like her had no chance in that hellhole, and I? I needed to eat."`
+			`	The color drains out of Deoric's face. "Monster." He says. "I will put an end to you. I will start a new family, as I should have before."`
+			`	Deoric lunges at Ildico, but before he can take more then a single step, Ildico's wrist flicks and her knife slams into Deoric's throat. He falls to the ground, bleeding out, dying.`
+			`	Ildico stands over him. "You are a coward. A miserable disgusting man." She spits on him, and pulls her dagger from his heaving neck. He tries to speak as his dying body squirms, but no words materialize and he goes still.`
+			`	She turns to you. "I am now a fugitive, and you are my only friend. I need you to get us out of here."`
+			`	Ildico walks back to corner far from any of the fighting, and retrieves a bundled baby you'd neglected to notice until now.`
+			choice
+				`	"You got it. Let's go."`
+				`	"Anything for you, Ildy."`
+				`	"Sure, but you owe me. Big."`
+				`	"No way. You're a murderer. I want nothing to do with you."`
+					goto leave
+				`	"Stop right there. I am putting you under citizen's arrest until the police arrive."`
+					goto arrest
+
+			`	She nods, her face softens, and tears glitter around her eyes. "Thank you, I knew- I knew I could count on you."`
+			`	Ildico recollects herself for a moment before pointing to a briefcase lying by a table. "Grab that, and let's go."`
+			choice
+				`	"Go where?"`
+
+			`	She sighs. "Not the Republic, nor the Free Worlds. After tonight I will be hunted for this. We need to go to <planet>. And you should probably not let anyone scan your ship until we get there."`
+			`	The two of you escape out into the night and onto your ship.`
+				launch
+
+			label arrest
+			`	Ildico puts her baby back down, and for just a second you see agony in her eyes. Killing her father or her arranged husband didn't break her mask, but somehow this does. A tear rolls down her cheek. She moves, too fast for you to react, and her dagger is in your throat.`
+			`	You lay on the ground, bleeding out, dying. Ildico standing above you, a broken expression on her face. "I thought I had one friend. What a fool I was. I'll never make that mistake again."`
+			`	She leaves you to die. The world grows dim, and hazy. You remember... you remember... someone...`
```

Comment:
> You remember... you remember... someone...

This seems unnecessarily vague, what is this supposed to convey?

Reply:
Neurons exploding in your brain as you die.